commit-template.txt

```diff
@@ -0,0 +1,11 @@
+topic: Short summary under 50 chars (if possible)
+
```

Comment:
How about quoting the seven rules for a good commit message from https://chris.beams.io/posts/git-commit/

IMHO at least using the imperative mood is missing here.

Reply:
Ok, will try to embed it here.